docs/EmptyState.md

```diff
@@ -21,9 +21,21 @@ import { EmptyState } from '@pxblue/react-components';
 
 | Prop Name   | Description                             | Type              | Required | Default |
 |-------------|-----------------------------------------|-------------------|----------|---------|
-| title       | The main text to display                | `string`          | yes      |         |
-| description | The secondary text to display           | `string`          | no       |         | 
-| icon        | The primary icon                        | `React.Component` | yes      |         |
 | actions     | Additional components to render below   | `React.Component` | no       |         |
+| classes     | Style overrides                         | `StyleRules`      | no       |         |    
```

Comment:
can you alphabetize these please?

Reply:
I kept the root at the top of the list, thoughts?

Reply:
yeah that's probably fine

Reply:
That's cool, but can we alphabetize the remainder?

Reply:
Yep, it's updated now.  I'm going to resolve this.